modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -332,3 +332,51 @@ Due to this, each node with the `:Person` label needs to be accessed and examine
 Due to the additional data accesses that the security checks need to do, this operation will be slower compared to executing the query as an unrestricted user.
 
 |===
+[[property-based-access-control-limitations]]
+=== Property-based access control limitations
+The same goes for when adding security rules based on property values. Similar to above we have a `restricted` and a `free` roles as an example:
```

Comment:
```suggestion
Extra node-level security checks are necessary when adding security rules based on property rules, and these can have a significant performance impact. In the examples below we use the roles `restricted` and a `free`:
```

I prefer not referencing the above section because the reader my want to look at this section in isolation